server/utils/services/tokenService.js

```diff
@@ -7,7 +7,7 @@ const tokenService = {
    * @param {Number} expiresIn - time for token to expire
    * @returns {String} token
    */
-  generateToken(payload, expiresIn) {
+  generateToken(payload, expiresIn = 60 * 20) {
```

Comment:
@chidioguejiofor why ?

Reply:
I thought it made sense to use a default expiry time